setup.py

```diff
@@ -28,9 +28,9 @@
     ],
     keywords='websocket client server trio',
     packages=find_packages(exclude=['docs', 'examples', 'tests']),
-    install_requires=['trio', 'trustme', 'wsproto'],
+    install_requires=['trio', 'wsaccel', 'wsproto'],
     extras_require={
-        'wsaccel': ['wsaccel'],
+        'dev': ['pytest', 'pytest-trio', 'trustme'],
```

Comment:
I'm not sure if it's the common pattern, but I've seen projects keep a pip `test-requirements.txt` file.  It's very explicit and descriptive and easily used by CI, etc.

Reply:
I found this person using the same `extras_require` approach: https://stackoverflow.com/a/28842733/122763